tst/SignalingApiFunctionalityTest.cpp

```diff
@@ -43,17 +43,15 @@ STATUS masterMessageReceived(UINT64 customData, PReceivedSignalingMessage pRecei
 
 STATUS signalingClientStateChanged(UINT64 customData, SIGNALING_CLIENT_STATE state)
 {
-    SignalingApiFunctionalityTest *pTest = (SignalingApiFunctionalityTest*) customData;
-
+    UNUSED_PARAM(customData);
```

Comment:
nice.. wondering if we should add a comment on the intent of this param on local contexts...